code/leslie/Python/lab07.py

```diff
@@ -0,0 +1,100 @@
+import math # because I'm using math.ceil() to round up the answer
```

Comment:
Great use of comments.

Letting people who read your code know what things are doing.